helmfile/charts/gatekeeper/podsecuritypolicies/values.yaml

```diff
@@ -40,6 +40,11 @@
 #             max: 65535
 #       allowedUnsafeSysctls:
 #         - ...
+#     # Custom values for user and group mutations
+#     mutation:
+#       runAsGroup: 999
+#       runAsUser: 999
+#       fsGroup: 999
 constraints: {}
 
 # List of pod labels that mutation should be exempted from
```

Comment:
This list can be dropped now right?

Reply:
I don't think I should right now, once we've implemented the rest and feel happy with how things work then we should do it.